tinygrad/state.py

```diff
@@ -67,6 +68,11 @@ def _rebuild_tensor_v2(storage, storage_offset, size, stride, requires_grad, bac
     if storage[2] not in offsets: return None
     byte_offset = offsets[storage[2]]+storage_offset*storage[1].itemsize
     ret = t[byte_offset:byte_offset+prod(size)].cast(storage[1])
+    # convert bfloat16 -> float16 using LLVM
+    # upstream LLaMA also does this conversion:
```

Comment:
So the one thing is that other model do use this, but I think its fine that we always convert bfloat16 to float16 for now, as we don't have great bfloat16 support.